client/src/api/payments.js

```diff
@@ -0,0 +1,11 @@
+import httpClient from './httpClient';
+
+export const pay = (projectId, userId, token, investmentAmount) => {
```

Comment:
similar to add projects route, the authorized shouldn't need to put in userId to make the payment because it will be handled by their auth token in the backend. This also reminds me that the edit profile may not need a req.params.id either as they should only be editing their own profile so I will make a separate ticket for that.